MAINTAINERS.md

```diff
@@ -49,10 +57,20 @@ or this project can be selected from Visual Studio and click Run.
 > nuget pack Minio.core.nuspec
 ... package built ...
 ```
+#### Upload the package to nuget.org
+Login to nuget.org(https://www.nuget.org) and find the API Key for Minio
+```powershell
```

Comment:
follow commands don't look like are for `powershell` starting with `export` - perhaps we can state that these instructions are for linux and change the markdown styling to `sh` instead?

Reply:
ok, could you try doing a restore first? ``` msbuild /p:Configuration=Release.Net452 /t:Restore ```
I will update the instructions.